src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/PetCreator.java

```diff
@@ -40,10 +43,12 @@
     private final Set<GQStateEnum> gqStatesToIgnore = new HashSet<GQStateEnum>();
     private GenomeLocSortedSet coverageLocSortedSet;
     private static String PET_FILETYPE_PREFIX = "pet_";
+    private static String PREFIX_SEPARATOR = "_";
     private static String REF_RANGES_FILETYPE_PREFIX = "ref_";
+    private BigQueryWriteClient bqWriteClient;
 
 
-    public PetTsvCreator(String sampleIdentifierForOutputFileName, String sampleId, String tableNumberPrefix, SAMSequenceDictionary seqDictionary, GQStateEnum gqStateToIgnore, final boolean dropAboveGqThreshold, final File outputDirectory, final CommonCode.OutputType outputType, final boolean writePetData, final boolean writeReferenceRanges) {
+    public PetCreator(String sampleIdentifierForOutputFileName, String sampleId, String tableNumber, SAMSequenceDictionary seqDictionary, GQStateEnum gqStateToIgnore, final boolean dropAboveGqThreshold, final File outputDirectory, final CommonCode.OutputType outputType, final boolean writePetData, final boolean writeReferenceRanges, final String projectId, final String datasetName) {
```

Comment:
Structurally it feels strange to have two ways of writing within this class, and then several other ways outside of this class.  If we feel pretty confident that this is going to work/scale then it doesn't matter too much since we'll clean this all up a ton (like all the other ways of writing).